circuit/src/circuit.rs

```diff
@@ -207,8 +206,9 @@ impl<
 
 		let bits2num_selector = Bits2NumChip::configure(&common, meta);
 		let n_shifted_selector = NShiftedChip::configure(&common, meta);
-		let is_zero_selector = IsZeroChip::configure(&common, meta);
-		let lt_eq = LessEqualConfig::new(bits2num_selector, n_shifted_selector, is_zero_selector);
+		let main_selector = MainChip::configure(&common, meta);
+		let main = MainConfig::new(main_selector);
+		let lt_eq = LessEqualConfig::new(bits2num_selector, n_shifted_selector, main.clone());
```

Comment:
Can we follow the convention and have chipset configs first, then selectors after. When defining constructors?
Apply in all cases

Reply:
Done in https://github.com/eigen-trust/protocol/pull/145/commits/948ef3472265ab9d1fd0cbbfcc08cd35154c0c1c

Reply:
I meant in the constructors, like so:
```rust
LessEqualConfig::new(main.clone(), bits2num_selector, n_shifted_selector)
```

Reply:
Thanks for the indication! @lazovicff 
Done in https://github.com/eigen-trust/protocol/pull/145/commits/b1e552983ab6faf31f59f2dbdb557ac0513e263e